doc/design/abs_backup.md

```diff
@@ -0,0 +1,68 @@
+# Backups using Azure Blob Service (ABS)
+
+In a very similar fashion to backups using [S3](./s3_backup.md), we want to have an option to store backups in ABS.
+
+## Setup
+
+When starting etcd operator, we need to provide flags in order to access the ABS storage account:
+```bash
+$ etcd-operator --backup-abs-secret ${secret_name} --backup-abs-container ${container_name} ...
```

Comment:
if both s3 and abs options are given, users can choose from s3 or abs? they are not mutual-exclusive, right?